tests/cli/test_cli_options.py

```diff
@@ -441,6 +492,12 @@ def main(args):
     # set working dir to script dir
     sc_dir = dirname(relpath(__file__))
 
+    parser = argparse.ArgumentParser(description='Test CLI Options')
+    parser.add_argument('-diff', help='show diffs when there is a test mismatch', action='store_true')
+    parser.add_argument('-apply', help='auto apply the changes from the results folder to the output folder', action='store_true')
```

Comment:
Usually, arguments are using their short notation `-d` or a complete word but with two dashes `--diff`. I haven't seen this mixture. You can also use this reference http://www.catb.org/esr/writings/taoup/html/ch10s05.html if you want to add short options that are commonly used and preserve expectations.